octopusdeploy/provider.go

```diff
@@ -94,6 +95,7 @@ func Provider() *schema.Provider {
 			"octopusdeploy_tag_set":                                        resourceTagSet(),
 			"octopusdeploy_team":                                           resourceTeam(),
 			"octopusdeploy_tenant":                                         resourceTenant(),
+			"octopusdeploy_tenant_project_environment":                     resourceTenantProjectEnvironment(),
```

Comment:
This is excellent. Every time customers try to use Terraform they want to break things down into individually deployed modules. Breaking relationships out into the most granular form is the best way to go in the future.